shop_stripe/migrations/0001_initial.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-04-07 20:52
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
I was waiting to see this magic show up.  Nothing you did, but the AUTH_USER_MODEL has been a wrinkle for a while in Django-dom.....